docker/actinia-core-alpine/Dockerfile

```diff
@@ -74,7 +74,7 @@ COPY --from=build /build/*.whl /build/
 # RUN apk add curl && curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && python3 get-pip.py pip==20.0.2 && rm get-pip.py
 RUN pip3 install /build/*
 # See in build stage why we need this here
-RUN pip3 install scikit-learn
+RUN apk add --repository http://dl-cdn.alpinelinux.org/alpine/edge/testing py3-scikit-learn
```

Comment:
In general, adding a package from edge to a stable alpine version (3.12) might lead to errors due to unmet dependencies, but in this case I guess it is fine.

Reply:
Thanks. I added a TODO for this